src/components/Grid/Grid.tsx

```diff
@@ -203,57 +188,9 @@ export const Grid: FC<GridProps> = ({
 
   const createArrow = useCallback(
     (elementId: string, pointerPosition: Position) => {
-      const isCreatingNewArrow = activeTool === ToolbarButtonType.CreateArrow;
       const isCreatingClassicArrow =
```

Comment:
Should we rename this to `isCreatingArrow`?

Reply:
jepp, egen pr